docs/server/README.md

```diff
@@ -23,11 +23,11 @@ Deployment
 1. Ensure you are on the latest version of the main branch, and run `gcloud app deploy --no-promote` from the app-engine directory. This deploys the web app to a sort of "staging environment", since web traffic continues to be directed to the previous version.
 2. When the deploy is done, visit the staging environment at `https://<VERSION_ID>-dot--<PROJECT_ID>.wl.r.appspot.com/` Get the versionID from the [Versions Console](https://console.cloud.google.com/appengine/versions). Manually ensure the app UX works as expected by scanning a QR code and starting a trip.
 3. Run the following two commands from graas/server/test:
-    - `NODE_PATH=../node/node_modules node post-vehicle-positions.js -u https://<version-id>-dot-lat-long-prototype.wl.r.appspot.com -a pr-test -e PR_TEST_ID_ECDSA`
+    - `NODE_PATH=../node/node_modules node test-position-updates.js -u https://<version-id>-dot-lat-long-prototype.wl.r.appspot.com -a pr-test -e PR_TEST_ID_ECDSA`
     - `NODE_PATH=../node/node_modules node post-service-alerts.js -u https://<version-id>-dot-lat-long-prototype.wl.r.appspot.com -a pr-test -e PR_TEST_ID_ECDSA`
```

Comment:
Please also add `post-stop-time-entities.js`, which is the unfortunately named test for posting trip-updates. Feel free to rename for the sake of sanity.

Reply:
@unreasonableman can you confirm that post-stop-time-entities.js is working for you? When I run the command 
` NODE_PATH=../node/node_modules node post-stop-time-entities.js -u https://lat-long-prototype.wl.r.appspot.com -a pr-test -e PR_TEST_ID_ECDSA ` from server/tests, I get a 404 error. I want to confirm it's good on your end before I investigate deeper (I did already delete & re-install node_modules)